deploy/crds/hypercloud.tmaxanc.com_v1alpha1_virtualmachinevolumeexport_s3_cr.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: hypercloud.tmaxanc.com/v1alpha1
+kind: VirtualMachineVolumeExport
+metadata:
+  name: s3-export
+spec:
+  virtualMachineVolume:
+    name: myrootdisk
+  destination:
+    s3:
+      # s3 endpoint with bucket and object name in virtual hosted style, https://bucketname.s3.region.amazonaws.com/objectkeyname
+      url: "http://ceph-bucket.rook-ceph-rgw-my-store.rook-ceph:80/disk.img"
+      # the name of secret which contains accessKeyID and secretAccessKey of the s3 endpoint
+      secretRef: ceph-bucket
```

Comment:
If `secretRef` refers to the secret generated by the `endpoint-secret.yaml`, the name should be the same as the name in the `endpoint-secret.yaml` file.

Reply:
Thank you for spotting! It's fixed now